chainer/functions/connection/embed_id.py

```diff
@@ -87,7 +87,7 @@ def backward(self, inputs, grad_outputs):
 def embed_id(x, W, ignore_label=None):
     """Efficient linear function for one-hot input.
 
-    This function implements so called *word embedding*. It takes two
+    This function implements so called *word embeddings*. It takes two
```

Comment:
In the other part, the word "word embeddings" is used. So I uses the word "word embeddings"